src/Propel/Runtime/ActiveQuery/ModelCriteria.php

```diff
@@ -1955,8 +1962,12 @@ public function doUpdate($updateValues, ConnectionInterface $con, $forceIndividu
                 // this criteria updates only one object defined by a concrete primary key,
                 // therefore there's no need to remove anything from the pool
             } else {
-                call_user_func([$this->modelTableMapName, 'clearInstancePool']);
-                call_user_func([$this->modelTableMapName, 'clearRelatedInstancePool']);
+                $modelTableMapName = $this->modelTableMapName;
+                if (!isset($modelTableMapName)) {
```

Comment:
should be !== to avoid cloaking call on a must-be-initialized var, and use simple strict null comparison.

Reply:
done